### ISSUE TITLE
Graph Viewer displays empty graph, with populated collections

### ISSUE THREAD

**Issue by cw00dw0rd** [arangodb/arangodb#9820]
## My Environment

* __ArangoDB Version__:        3.5

## Component, Query & Data

__Affected feature__:
Aardvark Graph Viewer

## Steps to reproduce

1. Create Named Graph with multiple definitions but only a portion of the collections currently contain data
2. Open Graph Viewer
3. `Graph: Your graph is empty. Click inside the white window to create your first node.` will most likely display or refresh until it does.

__Problem__:
When a user starts creating graphs in ArangoDB, if some of the collections don't contain data, the graph viewer will state `Graph: Your graph is empty. Click inside the white window to create your first node.`. This occurs even when there are correctly formatted edge and document collections, that on their own render a proper graph. This comes from the random selection of nodes in the graph viewer selecting an empty collection and then falsely reporting that the whole graph is empty. 

__Expected result__:
When a list of definitions contain multiple collections that are empty they should be ignored and the first collection with actual data should be used for the render.

**Comment by Simran-B** [comment]
A fix (https://github.com/arangodb/arangodb/pull/9827) will ship with v3.4.9 and v3.5.1

**Comment by maxkernbach** [comment]
Closing as fixed since version 3.5.1, which includes a fix for this issue, has been released.

From the [Changelog](https://github.com/arangodb/arangodb/blob/4290f381af33a6eb9f69b6cad47752b6af3e52d7/CHANGELOG#L222):
```
The graph viewer of the web interface now tries to find a vertex document of
all available vertex collections before it aborts.
```
